### ISSUE TITLE
Android build error version incompatibility on com.google.android.gms

### ISSUE THREAD

**Issue by duro** [tipsi/tipsi-stripe#295]
I have been trying to navigate through build errors on Android, which I assume are caused by another one of my dependencies wanting version 15.0.1 of the GMS package.

With no changes to what `react-native link` creates, I get this error:

```
Android dependency 'com.google.android.gms:play-services-maps' has different version for the compile (11.8.0) and runtime (15.0.1) classpath. You should manually set the same version via DependencyResolution
```

I have tried to force `tipsi-stripe` to use 15.0.1 for its `play-services-wallet` dependency with this:

```
compile (project(':tipsi-stripe')) {
    configurations.all {
        resolutionStrategy {
            force 'com.google.android.gms:play-services-wallet:15.0.1'
        }
    }
}
```

But then I get this error:

```
> Task :app:transformDexArchiveWithExternalLibsDexMergerForDebug FAILED
D8: Program type already present: android.support.design.widget.CoordinatorLayout$Behavior
```

Any idea where I go from here? I'm totally unable to get this to build.

**Comment by 82rules** [comment]
Any follow up on this issue I've run into the same incompatibility.

**Comment by theonetheycallneo** [comment]
@82rules @duro Posted a PR that fixed this with the latest stripe framework for both ios and android:

https://github.com/tipsi/tipsi-stripe/pull/311

**Comment by phuchuynhStrong** [comment]
Firstly, upgrade your `appcompat` and `design` to your `compileSdkVersion`. Then add `exclude group: "com.android.support" ` when add implementation project `tipsi-stripe`

**Comment by isnifer** [comment]
@duro we have merged in 5.5.0 this PR https://github.com/tipsi/tipsi-stripe/pull/350. See it
